runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -505,99 +509,72 @@ public void checkCompletion()
 
 					int tickRate = prediction.getProduce().getTickrate();
 					
-					if (offsetPrecisionMins == null || offsetTimeMins == null || (offsetPrecisionMins < tickRate && offsetPrecisionMins < 40) || prediction.getProduce() == Produce.WEEDS)
+					if (offsetPrecisionMins == null || offsetTimeMins == null || (offsetPrecisionMins < tickRate && offsetPrecisionMins < 40) || prediction.getProduce() == Produce.WEEDS
```

Comment:
as this is it won't notify if you watch the crop grow because it switches to a non-ticking state. It should give the notification while watching it too.

Reply:
It's using `Produce::tickrate`, which just provides the regular tick rate, as opposed to `PatchState::getTickRate` which is based on the current CropState

Reply:
the predictor uses PatchState's tickrate, so the done estimate will always be zero if its observed it done.